sequence_tagging_for_ner/ner_final.py

```diff
@@ -0,0 +1,241 @@
+
+
+def ner_net():
+    word = paddle.layer.data(name='word', type=d_type(word_dict_len))
+    #ws = paddle.layer.data(name='ws', type=d_type(num_ws))
+
+    word_embedding = paddle.layer.mixed(
+        name='word_embedding',
+        size=word_dim,
+        input=paddle.layer.table_projection(input=word, param_attr=emb_para))
+    #ws_embedding = paddle.layer.mixed(name='ws_embedding', size=caps_dim, 
+    #                    input=paddle.layer.table_projection(input=ws))
+    emb_layers = [word_embedding]  #[word_embedding, ws_embedding]
+
+    word_caps_vector = paddle.layer.concat(
+        name='word_caps_vector', input=emb_layers)
+    hidden_1 = paddle.layer.mixed(
+        name='hidden1',
+        size=hidden_dim,
+        bias_attr=std_default,
+        input=[
+            paddle.layer.full_matrix_projection(
+                input=word_caps_vector, param_attr=std_default)
+        ])
+
+    lstm_para_attr = paddle.attr.Param(initial_std=0.0, learning_rate=0.1)
+    hidden_para_attr = paddle.attr.Param(
+        initial_std=default_std, learning_rate=mix_hidden_lr)
+
+    lstm_1_1 = paddle.layer.lstmemory(
+        name='rnn1-1',
+        input=hidden_1,
+        act=paddle.activation.Relu(),
+        gate_act=paddle.activation.Sigmoid(),
+        state_act=paddle.activation.Sigmoid(),
+        bias_attr=std_0,
+        param_attr=lstm_para_attr)
+    lstm_1_2 = paddle.layer.lstmemory(
+        name='rnn1-2',
+        input=hidden_1,
+        act=paddle.activation.Relu(),
+        gate_act=paddle.activation.Sigmoid(),
+        state_act=paddle.activation.Sigmoid(),
+        reverse=1,
+        bias_attr=std_0,
+        param_attr=lstm_para_attr)
+
+    hidden_2_1 = paddle.layer.mixed(
+        size=hidden_dim,
+        bias_attr=std_default,
+        input=[
+            paddle.layer.full_matrix_projection(
+                input=hidden_1, param_attr=hidden_para_attr),
+            paddle.layer.full_matrix_projection(
+                input=lstm_1_1, param_attr=lstm_para_attr)
+        ])
+    hidden_2_2 = paddle.layer.mixed(
+        size=hidden_dim,
+        bias_attr=std_default,
+        input=[
+            paddle.layer.full_matrix_projection(
+                input=hidden_1, param_attr=hidden_para_attr),
+            paddle.layer.full_matrix_projection(
+                input=lstm_1_2, param_attr=lstm_para_attr)
+        ])
+
+    lstm_2_1 = paddle.layer.lstmemory(
+        name='rnn2-1',
+        input=hidden_2_1,
+        act=paddle.activation.Relu(),
+        gate_act=paddle.activation.Sigmoid(),
+        state_act=paddle.activation.Sigmoid(),
+        reverse=1,
+        bias_attr=std_0,
+        param_attr=lstm_para_attr)
+    lstm_2_2 = paddle.layer.lstmemory(
+        name='rnn2-2',
+        input=hidden_2_2,
+        act=paddle.activation.Relu(),
+        gate_act=paddle.activation.Sigmoid(),
+        state_act=paddle.activation.Sigmoid(),
+        bias_attr=std_0,
+        param_attr=lstm_para_attr)
```

Comment:
to be done,  haven't found a good way to rewrite, and fellow the original configuration temporarily